docs/sources/reference/components/pyroscope/pyroscope.java.md

```diff
@@ -84,13 +84,14 @@ The `profiling_config` block describes how async-profiler is invoked.
-`interval`    | `duration` | How frequently to collect profiles from the targets.                                                     | "60s"   | no
-`cpu`         | `bool`     | A flag to enable cpu profiling, using `itimer` async-profiler event.                                     | true    | no
-`sample_rate` | `int`      | CPU profiling sample rate. It is converted from Hz to interval and passed as `-i` arg to async-profiler. | 100     | no
-`alloc`       | `string`   | Allocation profiling sampling configuration  It is passed as `--alloc` arg to async-profiler.            | "512k"  | no
-`lock`        | `string`   | Lock profiling sampling configuration. It is passed as `--lock` arg to async-profiler.                   | "10ms"  | no
+Name          | Type       | Description                                                                                                                                                                                                                          | Default | Required
+--------------|------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|---------
+`interval`    | `duration` | How frequently to collect profiles from the targets.                                                                                                                                                                                 | "60s"   | no
+`cpu`         | `bool`     | A flag to enable cpu profiling, using `itimer` async-profiler event.                                                                                                                                                                 | true    | no
+`sample_rate` | `int`      | CPU profiling sample rate. It is converted from Hz to interval and passed as `-i` arg to async-profiler.                                                                                                                             | 100     | no
+`alloc`       | `string`   | Allocation profiling sampling configuration  It is passed as `--alloc` arg to async-profiler.                                                                                                                                        | "512k"  | no
+`lock`        | `string`   | Lock profiling sampling configuration. It is passed as `--lock` arg to async-profiler.                                                                                                                                               | "10ms"  | no
+`wall`        | `bool`     | Activates the `wall` mode when `cpu` profiling is enabled. This sample all threads equally every given period of time regardless of thread status: Running, Sleeping or Blocked. It is passed as `-e wall -t` arg to async-profiler. | false   | no
```

Comment:
```suggestion
`wall`        | `bool`     | Activates the `wall` mode when `cpu` profiling is enabled. This samples all threads equally every given period of time regardless of the thread status: Running, Sleeping, or Blocked. It is passed as an `-e wall -t` argument to the async-profiler. | false   | no
```

Should we move the extra information about `wall` to a spot below the table (like we do with other extended descriptions)?